app/src/androidTest/java/org/mozilla/fenix/ui/ReaderViewTest.kt

```diff
@@ -105,45 +104,46 @@ class ReaderViewTest {
 
     @Test
     fun verifyReaderViewToggle() {
-        // New three-dot menu design does not have readerview
-        if (!FeatureFlags.toolbarMenuFeature) {
-            val readerViewPage =
-                TestAssetHelper.getLoremIpsumAsset(mockWebServer)
+        // New three-dot menu design does not have readerview appearance menu item
+        val readerViewPage =
+            TestAssetHelper.getLoremIpsumAsset(mockWebServer)
 
-            navigationToolbar {
-            }.enterURLAndEnterToBrowser(readerViewPage.url) {
-                mDevice.waitForIdle()
-            }
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(readerViewPage.url) {
+            mDevice.waitForIdle()
+        }
 
-            readerViewNotification = ViewVisibilityIdlingResource(
-                activityIntentTestRule.activity.findViewById(R.id.mozac_browser_toolbar_page_actions),
-                View.VISIBLE
-            )
+        readerViewNotification = ViewVisibilityIdlingResource(
+            activityIntentTestRule.activity.findViewById(R.id.mozac_browser_toolbar_page_actions),
+            View.VISIBLE
+        )
 
-            IdlingRegistry.getInstance().register(readerViewNotification)
+        IdlingRegistry.getInstance().register(readerViewNotification)
 
-            navigationToolbar {
-                verifyReaderViewDetected(true)
-                toggleReaderView()
-                mDevice.waitForIdle()
-            }
+        navigationToolbar {
+            verifyReaderViewDetected(true)
+            toggleReaderView()
+            mDevice.waitForIdle()
+        }
 
+        if (!FeatureFlags.toolbarMenuFeature) {
             browserScreen {
                 verifyPageContent(estimatedReadingTime)
             }.openThreeDotMenu {
                 verifyReaderViewAppearance(true)
             }.closeBrowserMenuToBrowser { }
```

Comment:
... so that we are only running this specific piece of the test (the part that breaks with our new change) when the feature is ready :) That way, the rest of the test can still be run.